lib/TPP/Dialect/Tpp/TppOps.cpp

```diff
@@ -126,6 +126,27 @@ void BrgemmOp::build(OpBuilder &builder, OperationState &state,
   BrgemmOp::build(builder, state, inputs[0], inputs[1], output);
 }
 
+LogicalResult FusedBrgemmOp::verify() {
+  MemRefType tensorA = getBatchMatrixA().getType().cast<MemRefType>();
+  MemRefType tensorB = getBatchMatrixB().getType().cast<MemRefType>();
+  MemRefType matrixC = getMatrixC().getType().cast<MemRefType>();
+  if (!verifyBRGemmShape(tensorA, tensorB, matrixC))
```

Comment:
it would be good if `verifyBRGemmShape` emitted the error and this just propagated. It's hard to know what the shape error is when you have no extra info from the compiler. But this can be marked as TODO for later.